src/workerd/api/crypto/rsa.c++

```diff
@@ -225,6 +253,196 @@ SubtleCrypto::JsonWebKey Rsa::toJwk(KeyType keyType,
   return jwk;
 }
 
+kj::Maybe<AsymmetricKeyData> Rsa::fromJwk(KeyType keyType,
+                                          const SubtleCrypto::JsonWebKey& jwk) {
+  ClearErrorOnReturn clearErrorOnReturn;
+
+  if (jwk.kty != "RSA"_kj) return kj::none;
+  auto n = JSG_REQUIRE_NONNULL(jwk.n.map([](auto& str) { return str.asPtr(); }), Error,
+                               "Invalid RSA key in JSON Web Key; missing or invalid "
+                               "Modulus parameter (\"n\").");
+  auto e = JSG_REQUIRE_NONNULL(jwk.e.map([](auto& str) { return str.asPtr(); }), Error,
+                               "Invalid RSA key in JSON Web Key; missing or invalid "
+                               "Exponent parameter (\"e\").");
+
+  auto rsa = OSSL_NEW(RSA);
+
+  static constexpr auto kInvalidBase64Error = "Invalid RSA key in JSON Web Key; invalid base64."_kj;
+
+  // TODO(now): Use a decodeBase64Url variant
```

Comment:
Nit: I assume this refers to wanting to move code into a new decodeBase64Url() function, which is worth doing but not essential in this PR. TODO(now)s within the code base feel a bit scary to me as they indicate high urgency, perhaps this can be TODO(soon)?

LGTM otherwise.

Reply:
Comment is obsolete